mautic/mautic
# Video hits reports

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | y
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
Added new Video hits reports. 
Useful If your're using with tracked videos (https://github.com/mautic/documentation/pull/206)

#### Steps to test this PR:
1.  Apply PR
2. Go to reports and see new Videos > Video hits
3. Create reports and tests

#### List deprecations along with the new alternative:
1. 
2. 

#### List backwards compatibility breaks:
1. 
2.
